int64 x
int64 y
sensor_msgs/Image depth_image
bool flag